CHANGELOG.md

```diff
@@ -4,7 +4,10 @@
 * Migrated from dep to Go modules. Clients must now use the updated import path `github.com/stripe/veneur/v14`. Thanks, [andybons](https://github.com/andybons)!
 
 ## Added
+* The ability to emit statsd metrics from veneur-emit over plain TCP in addition to the current plain UDP. Thanks, [shrivu-stripe](https://github.com/shrivu-stripe)!
```

Comment:
You're right. I avoided bumping the version because I am not entirely sure what it entails. Now that all imports in go are "veneur/v14" does that mean we need to update every import to "v15"? Which files actually control the export artifacts version (is it just the dockerfile)?

I'll give it a shot now though and see how it goes.

Reply:
After looking at [this article](https://golang.org/doc/modules/major-version) it seems like bumping the major version does require explicitly updating import tags in all files. I guess the easiest way is to not bump the major version anymore unless there actually are backwards incompatible changes. Therefore I'm going to add a new version of "v14.1.0"